libraries/CurieSoftwareSerial/SoftwareSerial.cpp

```diff
@@ -230,9 +237,9 @@ SoftwareSerial::SoftwareSerial(uint32_t receivePin, uint32_t transmitPin, bool i
 {
   invertedLogic = inverse_logic;
   setTX(transmitPin);
-  txPin = transmitPin;
+  _transmitPin = transmitPin;
```

Comment:
txPin was originally a global variable in the cpp file 
i needed a variable that is an instance per each SoftwareSerial object.